README.md

```diff
@@ -26,7 +26,19 @@
 
 ## About hoop.dev
 
-Hoop.dev is an access gateway for databases and servers. Our AI-powered automations kill access policies and break-glass workflows with zero security compromises. How? Zero-config DLP policies with our AI Data Masking. Just-in-Time reviews by command and by access time. Hoop delivers security features like we are not even there for the developer. But present all the time for managers and auditors. That’s why great companies choose hoop.dev to provide secure access to anything. From databases to command line interfaces with Kubernetes or any server in any way.
+Hoop.dev is an access gateway for databases and servers with an API for packet manipulation. Because of the modern architecture powering Hoop, the open-source version includes advanced features like:
+
+ * Passwordless Auth, No Certificates: older gateways require high-maintenance certificate authorities. Hoop uses OIDC and Oauth2 for authentication, letting your IDP handle everything behind the scenes. Forget about certificates!
+ * Open-source SSO: support for Okta, Keycloak, Jumpcloud, and others. There is no need for Enterprise versions to integrate your own IDP. You're not limited to GitHub sign-in.
+ * Session recording: Linux, Docker, Kubernetes, Mysql, Postgres, MongoDB, and many more.
+ *	Just-in-time access: grant time-bound sessions using groups synced from your IDP.
+ * Slack and Teams Access Requests: Chatbot approval workflows can be done without leaving your chat app.
+
+Discover the unique capabilities that only Hoop can offer. From packet manipulation to web and proxy modes, Hoop is designed to meet your diverse needs.
+
+* Manipulate packets: Programmatically changes the gateway's environment and each connection's packets in real-time. Check out the Secrets Manager iration  e xample.
```

Comment:
"Check out the Secrets Manager iration e xample."
I'm not sure, but it is migration example?

```suggestion
* Manipulate packets: Programmatically changes the gateway's environment and each connection's packets in real-time. Check out the Secrets Manager migration example.
```